src/sagemaker/config/config_schema.py

```diff
@@ -89,13 +89,17 @@
 OBJECT = "object"
 ADDITIONAL_PROPERTIES = "additionalProperties"
 ENABLE_INTER_CONTAINER_TRAFFIC_ENCRYPTION = "EnableInterContainerTrafficEncryption"
+SESSION = "Session"
```

Comment:
This is a little confusing with sagemaker session, what will this be used for.? trying to understand if there is a better name for this.